test/CurveTests.ts

```diff
@@ -0,0 +1,88 @@
+import { ethers } from "hardhat";
+import { expect } from "chai";
+import { helperResetNetwork, helperSwapETHWith3CRV } from "./MainnetHelper";
+import { createMetapool, getMetapool, fundMetapool, setupMetapool, exchangeLvUSDfor3CRV, exchange3CRVfor3LvUSD } from "./CurveHelper";
+import { buildContractTestContext, ContractTestContext } from "./ContractTestContext";
+
+describe("CurveHelper Test Suite", function () {
+    let owner;
+    let user1;
+    let user2;
+    let r: ContractTestContext;
+    let lvUSD;
+    let token3CRV;
+
+    beforeEach(async function () {
+        // Reset network before integration tests
+        helperResetNetwork(14533286);
+        // Setup & deploy contracts
+        r = await buildContractTestContext();
+        lvUSD = r.lvUSD;
+        token3CRV = r.external3CRV;
+        [owner, user1, user2] = await ethers.getSigners();
+        // Mint 200 ETH of LvUSD for owner
+        await lvUSD.mint(owner.address, ethers.utils.parseEther("200.0"));
+        // Swap 200 ETH of 3CRV for owner
+        await helperSwapETHWith3CRV(owner, ethers.utils.parseEther("200.0"));
+    });
+
+    it("Initialize test funds", async function () {
+        // Make sure "owner" has funds needed for testing from "beforeEach" section
+        expect(await lvUSD.balanceOf(owner.address)).to.eq(ethers.utils.parseEther("200.0"));
+        expect(await token3CRV.balanceOf(owner.address)).gt(0);
+    });
+
+    it("Should create a 3CRV+LvUSD Metapool", async function () {
+        let poolAddress = ethers.constants.AddressZero;
+        poolAddress = await createMetapool(lvUSD, owner);
+        expect(poolAddress).to.not.equal(ethers.constants.AddressZero);
+    });
+
+    it("Should get values of a Metapool", async function () {
+        const token = lvUSD;
+        const pool = await getMetapool(await createMetapool(token, owner), owner);
```

Comment:
The `it `before already created a pool and the state is kept between `its` so no need to create again